app/views/events/show.html.erb

```diff
@@ -0,0 +1,43 @@
+<div class="row mt-4">
+  <div class="col-md-5 mb-2 ml-4">
+    <div class="card">
+      <div class="card-body">
+        <h5 class="card-title border-bottom pb-2"><%= @event.name %><small class="float-right">Owner: <%= @event.creator.name  %> </small></h5>
+        <p class="card-text"><%= @event.description %></p>
+        <p><strong>Location: </strong><small class="card-text"><%= @event.location %></small></p>
+        <p><strong>Date: </strong>
+        <small class="card-text">
+            <strong><%= @event.date.to_s.split(' ').first %> </strong>
+            <em><%= @event.date.to_s.split(' ')[1] %></em>
+        </small>
+        </p>
+      </div>
+    </div>
+  </div>
+
+    <div class="col-md-3 mb-2">
+    <div class="card">
+      <div class="card-body">
+        <h5 class="card-title border-bottom pb-2">List of Attendees</h5>
+        <% @event.attendees.each do |attendee|%>
+            <p><strong><%= attendee.name %></strong></p>
+        <% end %>
+
+      </div>
+    </div>
+  </div>
+
+   <div class="col-md-3 mb-2">
+    <div class="card">
+      <div class="card-body">
+        <h5 class="card-title border-bottom pb-2">Invite Users</h5>
+         <% @users.each do |user| %>
```

Comment:
- [ ] Think on how to refactor this into a helper method. Avoid loops, view logic, directly in views. Having the content coming from a helper method is faster than interpreting the content inside a block in the erb file.
Check[ this article about rails mistakes, especially the part about view logic.](https://www.toptal.com/ruby-on-rails/top-10-mistakes-that-rails-programmers-make#common-mistake-2-putting-too-much-logic-in-the-view)

Reply:
Done with changes